index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare class Log {
+  format(...data: any[]): string
+
+  debug(...data: any[]): void
+  info(...data: any[]): void
+  warn(...data: any[]): void
+  error(...data: any[]): void
+  fatal(...data: any[]): void
```

Comment:
```suggestion
  format(...data: unknown[]): string

  debug(...data: unknown[]): void
  info(...data: unknown[]): void
  warn(...data: unknown[]): void
  error(...data: unknown[]): void
  fatal(...data: unknown[]): void
```

You mostly always want `unknown` instead of `any`. The difference is that `unknown`, unlike `any`, MUST be narrowed to a more concrete type before it can be used.